### PR TITLE
Fix diverging parameter name and definitions

### PR DESCRIPTION
**Pull request by blarz**
Found by clang-tidy

### CODE REVIEW

**Comment by blarz** [comment]
Please discuss. Maybe some parameters should be named after how they are defined and vice versa.

File: src/frametree.h [hunk 1]
```diff
@@ -34,7 +34,7 @@ class FrameTree : public std::enable_shared_from_this<FrameTree> {
     bool focusClient(Client* client);
     //! focus a frame within its tree
     static void focusFrame(std::shared_ptr<HSFrame> frame);
-    bool focusInDirection(Direction dir, bool externalOnly);
```
Comment (dnnr) [inline src/frametree.h:37]: I think we agreed on camelCase as the canonical style (even though it's not consistently used yet)

File: src/utils.h [hunk 2]
```diff
@@ -87,7 +87,7 @@ void set_window_double_border(Display *dpy, Window win, int ibw,
 // returns the unichar in GSTR at position GSTR
 
 size_t      utf8_string_length(const std::string& str);
-std::string utf8_string_at(const std::string& str, size_t offset);
```
Comment (dnnr) [inline src/utils.h:90]: Looks to me like `offset` would be the better name here

**Comment by dnnr** [comment]
> Please discuss. Maybe some parameters should be named after how they are defined and vice versa.

I might find some more nitpicks, but for most of the cases, I mind the mismatch more than the naming. Except for my two comments, this looks good to me.

File: src/utils.h (hunk 2 above)
Comment (blarz) [reply src/utils.h]: Makes sense!

File: src/frametree.h (hunk 1 above)
Comment (blarz) [reply src/frametree.h]: Fixed

**Comment by blarz** [comment]
> > Please discuss. Maybe some parameters should be named after how they are defined and vice versa.
> 
> I might find some more nitpicks, but for most of the cases, I mind the mismatch more than the naming. Except for my two comments, this looks good to me.

Updated it. Yeah not all names are optimal, maybe other clang* tools will yell at us later.

**Comment by blarz** [comment]
'readability-inconsistent-declaration-parameter-name' could be added to the clang-tidy PR, now too

**Pull request merged**